frontend/amundsen_application/static/js/features/ColumnList/ColumnDetailsPanel/index.spec.tsx

```diff
@@ -0,0 +1,278 @@
+  getUniqueValues: () => mockStats,
+}));
+let mockLineageEnabled = true;
+jest.mock('config/config-utils', () => ({
+  isColumnListLineageEnabled: () => mockLineageEnabled,
+  getMaxLength: jest.fn(),
+}));
+
+describe('ColumnDetailsPanel', () => {
+  const setup = (propOverrides?: Partial<ColumnDetailsPanelProps>) => {
+    const props: ColumnDetailsPanelProps = {
+      columnDetails: mockColumnDetails,
+      togglePanel: jest.fn(),
+      ...propOverrides,
+    };
+    const wrapper = shallow(<ColumnDetailsPanel {...props} />);
+    return { props, wrapper };
+  };
+
+  describe('render', () => {
+    it('renders without issues', () => {
+      expect(() => {
+        setup();
+      }).not.toThrow();
+    });
+
+    it('triggers the panel toggle when the X button is clicked', () => {
+      const { props, wrapper } = setup();
+
+      wrapper.find('.btn-close').simulate('click');
+
+      expect(props.togglePanel).toHaveBeenCalled();
+    });
+
+    describe('renders copy column info buttons', () => {
+      it('renders two column info buttons', () => {
+        const { wrapper } = setup();
+
+        const actual = wrapper.find('.btn-default').length;
+        const expected = 2;
+
+        expect(actual).toEqual(expected);
+      });
+
+      it('first button copies column name', () => {
+        const { wrapper } = setup();
+
+        wrapper.find('.btn-default').first().simulate('click');
+
+        expect(navigator.clipboard.writeText).toHaveBeenCalledWith(
+          'column_name'
+        );
+      });
+
+      it('second button copies column link', () => {
+        const { props, wrapper } = setup();
+
+        wrapper.find('.btn-default').at(1).simulate('click');
+        const expected = getColumnLink(
+          props.columnDetails.tableParams,
+          props.columnDetails.name
+        );
+
+        expect(navigator.clipboard.writeText).toHaveBeenCalledWith(expected);
+      });
+    });
+
+    describe('renders column badges', () => {
+      it('should render badges', () => {
+        const { wrapper } = setup();
+
+        const actual = wrapper.find(BadgeList).length;
+        const expected = 1;
+
+        expect(actual).toEqual(expected);
+      });
+
+      it('should not render badges', () => {
+        const noBadgesColDetails = {
+          ...mockColumnDetails,
+          badges: [],
+        };
+        const { wrapper } = setup({ columnDetails: noBadgesColDetails });
+
+        const actual = wrapper.find(BadgeList).length;
+        const expected = 0;
+
+        expect(actual).toEqual(expected);
+      });
+    });
+
+    describe('renders a description', () => {
+      it('should render a description', () => {
+        const { wrapper } = setup();
+
+        const actual = wrapper.find(ColumnDescEditableText).length;
+        const expected = 1;
+
+        expect(actual).toEqual(expected);
+      });
+
+      it('should still render a description if only content is set', () => {
```

Comment:
Nit:
describe('when only content is set', () => {
   it('should still render a description', () => {